test/cets_SUITE.erl

```diff
@@ -1315,6 +1362,62 @@ disco_connects_to_unconnected_node(Config) ->
     cets_discovery:add_table(Disco, Tab),
     ok = cets_discovery:wait_for_ready(Disco, 5000).
 
+%% Joins from a bad (not fully connected) node
+%% Join process should check if nodes could contact each other before allowing to join
+joining_not_fully_connected_node_is_not_allowed(Config) ->
+    #{ct3 := Node3, ct5 := Node2} = proplists:get_value(peers, Config),
```

Comment:
Why not calling ct5 := Node5? Just to keep numbers matching the node number? Maybe it's easier that way to keep in mind the configuration of such node?

Reply:
Pid1, Pid5... hard to read test.
Given 2 nodes, we do something on first node Node1 and do something on the second node Node5 (surprise).